### ISSUE TITLE
.Flex-1 bug?

### ISSUE THREAD

**Issue by bsubedi26** [marklawlor/nativewind#243]
Hello, is the below expected?

In tailwind, the class `flex-1` sets the following css:
```css
  flex-grow: 1;
  flex-shrink: 1;
  flex-basis: 0%;
```

When doing a css output using: `npx tailwind -o output.css`. The styles does look correct since `flex: 1` is the same as above:
```css
.flex-1 {
  flex: 1;
}
```

However, when manually generating the stylesheet output using: `npx tailwindcss -i output.css --postcss postcss.config.js`
```js
"flex-1": {
  "flexGrow": 1,
  "flexShrink": 1
}
```

Shouldn't the generated stylesheet also include `flex-basis: 0%`?

**Comment by mateusz-ramski** [comment]
Its definitely broken. Check that flex-1 passed by className produces incorrect output. Flex-1 passed as standard style object works like expected.

```
<View>
            <View className='flex-row m-4'>
                <View className='flex-1 border border-black'>
                    <Text>Vulputate Ipsum</Text>
                </View>
                <View className='flex-1 border border-black'>
                    <Text>Vulputate Ipsum Justo Bibendum</Text>
                </View>
            </View>
            <View className='flex-row m-4'>
                <View className='border border-black' style={{ flex: 1 }}>
                    <Text>Vulputate Ipsum</Text>
                </View>
                <View className='border border-black' style={{ flex: 1 }}>
                    <Text>Vulputate Ipsum Justo Bibendum</Text>
                </View>
            </View>
        </View>
```

![Simulator Screen Shot - iPhone SE (3rd generation)](https://user-images.githubusercontent.com/65708055/191753930-529e60e8-be64-4dd7-a269-dabc5b60d5f0.png)

**Comment by mateusz-ramski** [comment]
Ok found issue: https://github.com/marklawlor/nativewind/pull/255

**Comment by marklawlor** [comment]
Fixed in `2.0.11`
